server/src/test/java/org/elasticsearch/discovery/ClusterDisruptionIT.java

```diff
@@ -72,7 +70,6 @@
 /**
  * Tests various cluster operations (e.g., indexing) during disruptions.
  */
-@ClusterScope(scope = Scope.TEST, numDataNodes = 0, transportClientRatio = 0, autoMinMasterNodes = false)
```

Comment:
I think this shouldn't be removed, although the `autoMinMasterNodes = false` can go.